test/test_hybrid.cpp

```diff
@@ -242,7 +242,7 @@ class HybridParsing : public testing::Test {
  */
 TEST_F(HybridParsing, GetTest) {
     // Create a new LibHTP transaction
-    htp_tx_t *tx = htp_connp_tx_create(connp);
+    htp_tx_t *tx = connp->in_tx =htp_connp_tx_create(connp);
```

Comment:
nit : missing space

Reply:
Whoops! I've pushed another patch for this

Reply:
thanks